x-pack/test/api_integration/apis/cases/privileges.ts

```diff
@@ -272,6 +293,36 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     }
 
+    for (const { user, owner } of [{ user: casesV3NoAssigneeUser, owner: CASES_APP_ID }]) {
+      it(`User ${
+        user.username
+      } with role(s) ${user.roles.join()} CANNOT change assignee`, async () => {
+        const caseInfo = await createCase(supertest, getPostCaseRequest({ owner }));
+        await updateCaseAssignee({
+          supertest: supertestWithoutAuth,
+          caseId: caseInfo.id,
+          assigneeId: caseInfo.created_by.profile_uid ?? '',
+          expectedHttpCode: 403,
+          auth: { user, space: null },
+        });
+      });
+    }
+
+    for (const { user, owner } of [{ user: casesV3ReadAndAssignUser, owner: CASES_APP_ID }]) {
```

Comment:
Could you please add some v2 users and verify that they can change assignees without any issues?